swap/src/bitcoin/wallet.rs

```diff
@@ -56,7 +57,7 @@ impl Wallet {
         let db = bdk::sled::open(wallet_dir)?.open_tree(SLED_TREE_NAME)?;
 
         let bdk_wallet = bdk::Wallet::new(
-            bdk::template::P2WPKH(private_key),
+            bdk::template::BIP84(key, KeychainKind::External),
             None,
```

Comment:
Hmm, I guess we don't need a change descriptor because we always swap everything... Thoughts?

Reply:
Yeh that was what I thought. In the case we somehow had change I assumed it would default to the standard descriptor to derive a change address, but I could not confirm this. I will explicitly specify a change address so we can be sure of the behaviour in case we end up having change somehow.